src/Tests/Html2PdfTest.php

```diff
@@ -2,13 +2,22 @@
 
```

Comment:
It looks to me like there are some tests in here that don't have assertions?

Reply:
They don't assert anything but they do expect exceptions? What do we do in this case - they seem like poorly written tests with the testSecurityGood test not asserting anything before I added assertTrue(true) 😂

Reply:
Fair enough.